excercises/transformation/SDLProject/solution/solution.cpp

```diff
@@ -0,0 +1,158 @@
+#define GL_SILENCE_DEPRECATION
+#define GL_GLEXT_PROTOTYPES 1
+#define LOG(argument) std::cout << argument << '\n'
+
+#ifdef _WINDOWS
+#include <GL/glew.h>
+#endif
+
+#include <SDL.h>
+#include <SDL_opengl.h>
+#include "glm/mat4x4.hpp"
+#include "glm/gtc/matrix_transform.hpp"
+#include "ShaderProgram.h"
+
+
+const char V_SHADER_PATH[] = "shaders/vertex.glsl",
```

Comment:
And remember to add this summer's stylistic changes later! This is not a priority for tomorrow, though.